### DISCUSSION TITLE
Novel Features

### DISCUSSION THREAD

**Discussion by Tiham2001** [Tiham2001/tarp#9] [General]
How do we get the ranking of the universities? Like on the basis of which factors will we decide?

**Comment by annanya1803** [comment]
We can search for the universities on google and their rankings. We can rank them on the basis of their good result rate and their placement ranking.

**Comment by Pranav400-source** [comment]
> How do we get the ranking of the universities? Like on the basis of which factors will we decide?

By using websites like usnews.com or yocket.com

**Reply by Tiham2001** [reply]
Yes that's a good idea or we can even follow the QS World Rankings

**Reply by shalinimishra31** [reply]
Yes, these websites are credible.

**Reply by Tiham2001** [reply]
Okay🙂
